gobblin-runtime/src/main/java/org/apache/gobblin/runtime/mapreduce/MRJobLauncher.java

```diff
@@ -832,6 +871,10 @@ public void execute() throws IOException {
           LOG.info("Adding additional commit step");
           gobblinMultiTaskAttempt.addCleanupCommitStep(cleanUpCommitStep);
         }
+        // Finishing of Mapper declares the actually execution finished, safe to report 1.0f.
+        if (customizedProgressEnabled) {
```

Comment:
MR framework already does this. So remove this to avoid confusion since the non-customized code is not treated differently.

Reply:
Will address.